src/interface/.gitignore

```diff
@@ -1,3 +0,0 @@
-gen-*
```

Comment:
why remove it？

Reply:
Its contents has been moved to the .gitignore file in the project root directory.